src/de/gurkenlabs/litiengine/graphics/animation/AsepriteHandler.java

```diff
@@ -0,0 +1,412 @@
+   *
+   * @param spritesheetResource the animation object to export
+   * @throws ExportAnimationException if the export fails
+   */
+  public static String exportAnimation(SpritesheetResource spritesheetResource) throws ExportAnimationException {
+    String json = createJson(spritesheetResource);
+    return json;
+  }
+  
+  /**
+   * Creates the json representation of an animation object and returns it as a string.
+   *
+   * @param spritesheetResource spritesheetResource object to export as json.
+   * @return the json as a string.
+   * @throws ExportAnimationException if the export fails
+   */
+  private static String createJson(SpritesheetResource spritesheetResource) throws ExportAnimationException {
+    Spritesheet spritesheet = Resources.spritesheets().load(spritesheetResource);
+    assert spritesheet != null;
+    int[] keyframes = Resources.spritesheets().getCustomKeyFrameDurations(spritesheet);
+    Frames[] frames = new Frames[keyframes.length];
+    
+    if (frames.length != spritesheet.getTotalNumberOfSprites()) {
+      throw new ExportAnimationException("Different dimensions of keyframes and sprites in spritesheet");
+    }
+    
+    // Build the frames object in the json
+    int numCol = spritesheet.getColumns();
+    int numRows = spritesheet.getRows();
+    int frameWidth = spritesheet.getSpriteWidth();
+    int frameHeight = spritesheet.getSpriteHeight();
+    
+    for (int i = 0; i < numRows; i++) {
+      for (int j = 0; j < numCol; j++) {
+        final int row = i;
+        final int col = j;
+        Map<String, Integer> frame = new HashMap<String, Integer>() {{
+          put("x", (0 + col * frameWidth));
+          put("y", (0 + row * frameHeight));
+          put("w", frameWidth);
+          put("h", frameHeight);
+        }};
+        Map<String, Integer> spriteSourceSize = new HashMap<String, Integer>() {{
+          put("x", 0);
+          put("y", 0);
+          put("w", frameWidth);
+          put("h", frameHeight);
+        }};
+        Map<String, Integer> sourceSize = new HashMap<String, Integer>() {{
+          put("w", frameWidth);
+          put("h", frameHeight);
+        }};
+        int duration = keyframes[i + j];
+        String index = String.valueOf(i + j);
+        frames[i + j] = new Frames("frame " + index,
+                frame,
+                false,
+                false,
+                spriteSourceSize,
+                sourceSize,
+                duration);
+      }
+    }
+    
+    // Build the meta object in the json
+    int spritesheetWidth = frameWidth * numCol;
+    int spritesheetHeight = frameHeight * numRows;
+    Map<String, Integer> size = new HashMap<String, Integer>() {{
+      put("w", spritesheetWidth);
+      put("h", spritesheetHeight);
+    }};
+    String spritesheetName = spritesheet.getName();
+    Layer[] layers = {new Layer("Layer", 255, "normal")};
+    Meta meta = new Meta("http://www.aseprite.org/",
```

Comment:
Please use [links.properties](https://github.com/gurkenlabs/litiengine/blob/master/utiliti/localization/links.properties) for external links. Also, I don't really like the in-line version information here. Maybe make that reusable.